internal/website/content/howto/sql/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "MySQL/PostgreSQL"
+date: 2019-06-21T09:26:56-07:00
+showInSidenav: true
+---
+
+Connecting to cloud providers' hosted database services require additional steps
+to ensure the security of the connection. The Go CDK makes opening these
```

Comment:
It would be nice to spend a bit more ink here to explain what these extra steps are that are being saved. In other words, I feel that the elevator pitch for these APIs is insufficient in its current form :)

Reply:
Took a pass, LMK if you're looking for more.